src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -118,10 +119,14 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
                 .orElse(personToEdit.getExpectedGraduationYear());
         Rating updatedRating = personToEdit.getRating();
         Resume updatedResume = editPersonDescriptor.getResume().orElse(personToEdit.getResume());
+
+        // Doesn't allow editing of interview date
+        InterviewDate interviewDate = personToEdit.getInterviewDate();
+
         Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
 
         return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress,
-                updatedExpectedGraduationYear, updatedRating, updatedResume, updatedTags);
+                updatedExpectedGraduationYear, updatedRating, updatedResume, interviewDate, updatedTags);
```

Comment:
Since `Rating` and `InterviewDate` is not updated here, do you think we should drop "updated" for Rating? @kexiaowen

Reply:
Sure!